seed/models/properties.py

```diff
@@ -483,7 +503,10 @@ def record_dict(log):
             filename = file = None
             if log.import_filename:
                 filename = path.basename(log.import_filename)
-                file = settings.MEDIA_URL + '/'.join(log.import_filename.split('/')[-2:])
+                if settings.USE_S3 is True:
+                    file = create_presigned_url(settings.AWS_STORAGE_BUCKET_NAME, log.import_filename)
```

Comment:
Because the second argument to create_presigned_url is meant to be the 'key', which is the full path to file eg: media/uploads/'file_uploaded', whereas filename in the case will just be the name of the file which is not what we want

Reply:
Ah okay, thanks for the explanation!